packages/linux-drivers/mk_arcade_joystick_rpi/package.mk

```diff
@@ -0,0 +1,70 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+#
+#  OpenELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  OpenELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="mk_arcade_joystick_rpi"
+PKG_VERSION="0a70997"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://github.com/Turro75/mk_arcade_joystick_rpi"
+PKG_URL="$PKG_SITE.git"
+PKG_SOURCE_DIR="${PKG_NAME}-${PKG_VERSION}"
+PKG_DEPENDS_TARGET="toolchain linux"
+PKG_NEED_UNPACK="$LINUX_DEPENDS"
+PKG_SECTION="driver"
+PKG_SHORTDESC="test building for RPI4"
+PKG_TOOLCHAIN="manual"
+
+PKG_IS_ADDON="no"
+PKG_AUTORECONF="no"
+
+if [ "$TARGET_KERNEL_ARCH" = "arm64" -a "$TARGET_ARCH" = "arm" ]; then
+  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET gcc-linaro-aarch64-elf:host"
+  export PATH=$TOOLCHAIN/lib/gcc-linaro-aarch64-elf/bin/:$PATH
+  TARGET_PREFIX=aarch64-elf-
+fi
+
+MYFLAG=""
+case $DEVICE in
```

Comment:
Need to handle case for RPi and other Devices/Projects or do a check in the project to enable it for only RPi4.

Similar to here (in RPi Project) for the RPi4 options:
https://github.com/libretro/Lakka-LibreELEC/blob/73ddb29abca9eedd58f155b66f76b25d66192676/projects/RPi/options#L128

Reply:
Thanks for the point, I took profit by removing the various ifdef so it can be compiled without any specific -D on every rpi.
The module now detects the rpi at loading time and adjust the base address automatically.
So it compiles and runs on every rpi. Due to missing hardware I tested it only on pi4 and pi3b+. 

just read the message from Ntemis, ok I wait for coming upgrade.
In the meanwhile I close the PR.